command.go

```diff
@@ -351,6 +354,26 @@ func parseLoadArgs(args []string) (Command, error) {
 	return l, nil
 }
 
+func parsePasswdArgs(args []string) (Command, error) {
+	flag := NewFlagSet("vaulted passwd")
+	err := flag.Parse(args)
+	if err != nil {
+		return nil, err
+	}
+
+	if flag.NArg() < 1 {
+		return nil, ErrNotEnoughArguments
+	}
+
+	if flag.NArg() > 1 {
+		return nil, ErrTooManyArguments
+	}
+
+	p := &Passwd{}
+	p.VaultName = flag.Arg(0)
```

Comment:
Because of how similar (exactly) this is to Copy I think I might lean towards:

c := &Copy{}
c.OldVaultName = flag.Arg(0)
c.NewVaultName = flag.Arg(0)